corehq/messaging/scheduling/views.py

```diff
@@ -776,8 +778,12 @@ def parent_pages(self):
     @property
     def page_context(self):
         context = super().page_context
+        case_count = 0
```

Comment:
nit: I understand that even 0 would work, though I would say `None` is better. 0 as case counts implies that there are 0 cases whereas `None` means we don't know it, and I feel latter is actually true.

Reply:
c7acbc8a9a1